db/migrate/20171019014656_create_vendor.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVendor < ActiveRecord::Migration
+  def change
+    create_table :vendors do |t|
+      t.string :name
+      t.string :street
+      t.string :city
+      t.string :state
+      t.string :zipcode
+      t.string :status
+      t.text :comments
```

Comment:
missing `t.timestamps` - would that be useful?

Reply:
Hey it's @pcai ! Cool
timestamps are a good idea.